uiowa/uiowa
# Sticky indicated cards

Resolves #6695




<img width="649" alt="Screenshot 2024-11-01 at 3 50 56 PM" src="https://github.com/user-attachments/assets/30f1c5be-eb9a-4c45-a81f-446a06b6427d">


# How to test

- `ddev blt frontend` - ignore the depreciations/warnings. this is to account for uids_base's switch from node-sass to sass.
- Mark one or more content nodes as sticky.
- Using any view that has a sticky sort on it, see that the content is sorted as such and has the new UIDS indicator using the pre-title slot.
- Using VoiceOver, traverse the non-tab indexed content with ctrl+option+right arrow. VoiceOver should read out "Pinned content, custom sorted."
- Go in to that view and remove the sort sticky. Save the view, clear cache and reload the placed view. There should be no sticky indicators. Even though the content was marked as sticky globally, in this context it is not.
- Generally check other styles to make sure there are no regressions with the way the uids_base SCSS files are compiled.

# Before Merge

- update uids hash
